gameci-1_intro.md

```diff
@@ -0,0 +1,146 @@
+You may refer to the [complete workflow on GitHub](https://github.com/finol-digital/Card-Game-Simulator/blob/develop/.github/workflows/main.yml), but going forward, I will excerpt the relevant code here.
+
+## The Code
+
+Let's get started with the start of my workflow:
+
+{% highlight yml %}
+{% raw %}
+# .github/workflows/main.yml
+name: Test, Build, and Deploy with GameCI
+on:
+  push:
+    branches:
+      - develop
+    paths:
+      - 'Assets/**'
+      - 'Packages/**'
+      - 'ProjectSettings/**'
+  pull_request:
+    types:
+      - opened
+    branches:
+      - main
+    paths:
+      - 'Assets/**'
+      - 'Packages/**'
+      - 'ProjectSettings/**'
+  release:
+    types:
+      - published
+  workflow_dispatch:
+    inputs:
+      workflow_mode:
+        description: '[release] [Android, iOS, StandaloneLinux64, WebGL, StandaloneWindows, StandaloneWindows64, WSAPlayer, StandaloneOSX, Steam]'
+        required: false
+        default: ''
+jobs:
+{% endraw %}
+{% endhighlight %}
+
+## Trigger On
+
+As you can see, there are 4 triggers on: `push` to develop, `pull_request` to main, `release` published (with GitHub Release), and `workflow_dispatch`.
+
+The `push` and `pull_request` triggers should run checks to validate each commit, along with a final validation before merging changes to `main`.
+Runs on `push` and `pull_request` should only trigger if a file has changed in the actual Unity project, ie in `Assets/`, `Packages/`, or `ProjectSettings/`.
+Any changes outside of these folders would not cause our Unity project to change, so they can be ignored.
+For example, if we modify the README.md for our project, we wouldn't want the workflow to run, since it wouldn't actually be testing any relevant change to the Unity project.
+
+Runs on `release` will trigger whenever a release is created in the GitHub UI, and we can use `if: github.event.action == 'published'` to detect this scenario.
+
+The triggers for `workflow_dispatch` are the most interesting, since that's where we have setup the most control over the workflow.
+For example, if we wanted to create 2 Windows .exe's to download and run, we can simply pass `StandaloneWindows StandaloneWindows64` as the input.
+This would create both 32-bit and 64-bit Windows executables and upload them to GitHub, where they could then be downloaded for testing.
+
+Taking it a step further, we could input `release Steam` to run the entire production deployment pipeline for Steam.
+My Steam depots involve 4 artifacts: 1) Windows 32-bit, 2) Windows 64-bit, 3) Linux, and 4) Mac.
+Building and deploying these artifacts is split across multiple jobs, so it would now be good to get a high-level overview of all the jobs.
+
+## The Jobs
+
+Going into detail on each job would make this article too long.
+Furthermore, a reader who only wants to publish Android builds on Google Play would be interested in a different set of jobs than a reader who wants to publish PC games on Steam.
```

Comment:
```suggestion
Furthermore, people who only wants to publish Android builds on Google Play would be interested in a different set of jobs than people who wants to publish PC games on Steam.
```

Think of who your readers are. Use something more specific than a reader or use generic `people`. Otherwise you're asking the reader to see how you write from your (own) perspective.

Reply:
Generally I would recommend to not make excuses and remove this whole text and start with the real information: `Here is a quick overview of...`

The excuse is 100% valid, but you don't have to tell your readers because it doesn't bring them any value.